Sources/SwiftParser/Lexer.swift

```diff
@@ -97,13 +97,17 @@ public struct Lexer {
       SyntaxText(baseAddress: start.advanced(by: leadingTriviaByteLength+textByteLength),
                  count: trailingTriviaByteLength)
     }
+
+    public var debugDescription: String {
+      return String(syntaxText: SyntaxText(baseAddress: start, count: byteLength))
```

Comment:
It’s a good start. Maybe we can render the flagset here. And even dump the trivia.

Reply:
The flags are still being rendered if you `po currentToken` in the parser. This is what the debug output looks like with this change.
```
(lldb) po self.currentToken
▿ func 
  - tokenKind : SwiftSyntax.RawTokenKind.funcKeyword
  ▿ flags : Flags
    - rawValue : 0
  ▿ start : 0x0000000104014000
    - pointerValue : 4362158080
  - leadingTriviaByteLength : 0
  - textByteLength : 4
  - trailingTriviaByteLength : 1

```